curation_concerns-models/app/jobs/ingest_local_file_job.rb

```diff
@@ -22,16 +22,18 @@ def run
 
     if actor.create_content(File.open(path), filename, mime_type(filename))
       FileUtils.rm(path)
-      CurationConcerns.queue.push(ContentDepositEventJob.new(generic_file.id, user_key))
 
-      message = "The file (#{File.basename(filename)}) was successfully deposited."
-      subject = 'Local file ingest'
+      # send message to user on import success
+      if CurationConcerns.config.respond_to?(:after_import_local_file_success)
```

Comment:
I would expect this PR to add `CurationConcerns.config.after_import_local_file_success` and for it to call `CurationConcerns.queue.push ...` like what was deleted on line 25.